src/Components/OffenderBreadcrumb/OffenderBreadcrumb.test.js

```diff
@@ -1,11 +1,19 @@
 import React from 'react'
-import { shallow } from 'enzyme'
+import ShallowRenderer from 'react-test-renderer/shallow'
```

Comment:
Curious to know why you've switched enzyme out.

Reply:
Jamie suggested that we should be moving away from `enzyme` and moving to `react-test-renderer` where possible.

I've been a bit inconsistent though within this PR, mostly due to limitations with the shallow rendering in `react-test-renderer`.
 * It doesn't call `onComponentMount`  
 * It only renders the top most element and you can't dive, like in enzyme.
 
I'm not totally convinced that there is a one size fits all approach to this.